libnavigation-ui/src/main/java/com/mapbox/navigation/ui/map/NavigationMapboxMap.java

```diff
@@ -236,9 +236,11 @@ public void clearMarkers() {
    * @param location to update the icon and query the map
    */
   public void updateLocation(Location location) {
```

Comment:
As this is a `public` method, could we add `@NonNull` / `@Nullable` annotations for Kotlin users 😅?